packages/bot/src/utils/notifications/bot-notifications.js

```diff
@@ -0,0 +1,9 @@
+import { localize } from 'deriv-translations/lib/i18n';
+
+export const switch_account_notification = {
+    key          : 'bot_switch_account',
+    header       : localize('You have switched accounts.'),
+    message      : localize('Our system will end any DBot trades that are running, and DBot will not place any new trades.'),
```

Comment:
To me this message implies that a trade will be interrupted when switching accounts, can we say something like `Our system will finish any running DBot trades, and DBot will not place any new trades.`

Reply:
@aaron-binary we got the message from binary-bot and has been checked it with yolanda